### PR TITLE
IGAPP-989: Backbutton on map hidden 

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Test on real small devices (height < 650)
1. http://localhost:9000/testumgebung/de/locations
2. select a poi on the map
3.  bottom action sheet should open as far as the map icons are visible

### CODE REVIEW

File: web/src/styles/BottomActionSheet.css [hunk 1]
```diff
@@ -16,5 +16,5 @@
 }
 
 [data-rsbs-has-header='false'] [data-rsbs-header] {
-  padding-top: calc(24px + env(safe-area-inset-top));
+  padding-top: calc(32px + env(safe-area-inset-top));
```
Comment (f1sh1918) [inline web/src/styles/BottomActionSheet.css:19]: Without a title in the header (details view) the bottomActionSheet handle becomes difficult to grab --> increased padding

File: web/src/components/Collapsible.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ const CollapsibleHeader = styled.div`
   display: flex;
   justify-content: space-between;
   cursor: pointer;
+  outline: none;
```
Comment (f1sh1918) [inline web/src/components/Collapsible.tsx:21]: rm ugly outline when clicking on collapsible header

Comment (klinzo) [reply web/src/components/Collapsible.tsx:21]: When was this shown? Can't reproduce or maybe I am just blind :eyes:

File: web/src/components/MapView.tsx [hunk 3]
```diff
@@ -117,6 +117,11 @@ const MapView = React.forwardRef((props: MapViewProps, ref: React.Ref<MapRef>):
     [changeSnapPoint, selectFeature]
   )
 
+  const onBackButton = () => {
```
Comment (klinzo) [inline web/src/components/MapView.tsx:120]:
```suggestion
  const deselectFeature = () => {
```

File: web/src/utils/getSnapPoints.ts [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
+
+// Calculates detail snapPoint by reducing height with navigation and iconHeight on the map
+const getDetailSnapPoint = (maxHeight: number) =>
+  maxHeight - dimensions.headerHeightLarge - dimensions.navigationMenuHeight - mapIconsHeight
+
 export const getSnapPoints = (maxHeight: number): number[] => [
   maxHeight * min,
   maxHeight * mid,
-  maxHeight * max,
+  getDetailSnapPoint(maxHeight),
```
Comment (klinzo) [inline web/src/utils/getSnapPoints.ts:14]: I think you could just inline this with a comment, no need to extract to a function. But fine like it is :)

**Review by klinzo — APPROVED** [review]
Tested on firefox, nice :tada:

**Review by SteffiStuffel — APPROVED** [review]

**Comment by SteffiStuffel** [comment]
Tested on Chrome

File: web/src/components/Collapsible.tsx (hunk 2 above)
Comment (f1sh1918) [reply web/src/components/Collapsible.tsx:21]: click on the collapseHeader and open content / close

**Pull request merged**